projects/storefrontlib/layout/a11y/keyboard-focus/autofocus/auto-focus.service.ts

```diff
@@ -29,7 +29,10 @@ export class AutoFocusService extends EscapeFocusService {
   /**
    * Indicates whether any of the focusable child elements is focused.
    */
-  hasPersistedFocus(host: HTMLElement, config: PersistFocusConfig): boolean {
+  hasPersistedFocus(
+    host: HTMLElement | null | undefined,
```

Comment:
I removed undefined from the type, and it still works. Why is it here?

Reply:
that's becuse `host` in `findFirstFocusable` is `HTMLElement | null | undefined`.

Reply:
makes sense! it ties to this here https://github.com/SAP/spartacus/pull/15658#discussion_r870762291 since we're saying to put | undefined instead